.github/actions/cache-build/action.yml

```diff
@@ -16,5 +16,5 @@ runs:
     - name: build gradle
       uses: gradle/gradle-build-action@40b6781dcdec2762ad36556682ac74e31030cfe2 # v2.5.1
       with:
-        arguments: ${{ inputs.arguments }}
+        arguments: ${{ inputs.arguments }} --info --stacktrace
```

Comment:
Is this intentional?

Reply:
Yes, because before that, all errors had to be investigated manually, gradle hides a lot of detail in the logs without these flags.